### PR TITLE
Feature/podcast

### PR DESCRIPTION
**Pull request by Majid-pkz**

### CODE REVIEW

**Comment by Majid-pkz** [comment]
I created my own account for Buzzsprout to get the id and everything. So Obviously that part needs NDA podcast id.

File: src/app/components/podcast/Buzzsprout.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      container.appendChild(script);
+    }
+  }, [scriptSrc, containerId]);
+
+  return (
+    <div className={styles.podcastContainer}>
+      <div className={styles.podcastHeader}>
+        <Typography variant={TypographyVariant.H2}> Podcasts</Typography>
+      </div>
+      <div id={containerId}></div>;
```
Comment (bhumkarpratik) [inline src/app/components/podcast/Buzzsprout.tsx:31]: I guess we missed the semi-colon, we can delete this before merge..

**Review by bhumkarpratik — APPROVED** [review]
This is perfect!

**Comment by Majid-pkz** [comment]
> On the mobile view, the size of the window should be similar to the size of articles. but, considering everything else, this is perfect!

fixed

File: src/app/components/podcast/Buzzsprout.tsx (hunk 1 above)
Comment (Majid-pkz) [reply src/app/components/podcast/Buzzsprout.tsx]: Fixed

**Comment by Majid-pkz** [comment]
> On the mobile view, the size of the window should be similar to the size of articles. but, considering everything else, this is perfect!

fixed

**Review by bhumkarpratik — APPROVED** [review]
LGTM

**Pull request merged**